diff -Naur a/src/csd/libappletdecoration/previewclient.cpp b/src/csd/libappletdecoration/previewclient.cpp
--- a/src/csd/libappletdecoration/previewclient.cpp	2020-11-09 18:53:22.000000000 +0000
+++ b/src/csd/libappletdecoration/previewclient.cpp	2021-02-16 12:56:50.271000000 +0000
@@ -426,8 +426,14 @@
     setKeepBelow(!isKeepBelow());
 }
 
+#if KDECORATION2_VERSION_MINOR >= 20
+void PreviewClient::requestShowWindowMenu(const QRect &rect)
+{
+    Q_UNUSED(rect);
+#else
 void PreviewClient::requestShowWindowMenu()
 {
+#endif
     emit showWindowMenuRequested();
 }
 
diff -Naur a/src/csd/libappletdecoration/previewclient.h b/src/csd/libappletdecoration/previewclient.h
--- a/src/csd/libappletdecoration/previewclient.h	2020-11-09 18:53:22.000000000 +0000
+++ b/src/csd/libappletdecoration/previewclient.h	2021-02-16 12:55:40.195000000 +0000
@@ -119,7 +119,11 @@
     void requestToggleKeepAbove() override;
     void requestToggleKeepBelow() override;
     void requestToggleShade() override;
+#if KDECORATION2_VERSION_MINOR >= 20
+    void requestShowWindowMenu(const QRect &rect) override;
+#else
     void requestShowWindowMenu() override;
+#endif
     void requestShowApplicationMenu(const QRect &rect, int actionId) override;
     void requestToggleOnAllDesktops() override;
 
